pkg/build/controller/build/pluginconfig/config.go

```diff
@@ -3,64 +3,51 @@ package pluginconfig
 import (
 	"io/ioutil"
 
-	"github.com/golang/glog"
+	"github.com/ghodss/yaml"
```

Comment:
hrm, is this new yaml? missed this one

Reply:
> hrm, is this new yaml? missed this one

We use inside the library that was previously being called.